src/OJP.Mod

```diff
@@ -388,6 +388,10 @@ MODULE OJP; (*Oberon compiler for JVM in Oberon-07*)
           ELSIF (x.type.form = OJB.Int) & (y.type.form = OJB.Real) THEN
             OJG.RealToInt(y);
             restyp := x.type
+          ELSIF ((x.type.form = OJB.Pointer) & (y.type.form = OJB.Pointer))
```

Comment:
There is an extra set of parenthesis in the ELSIF condition, it should be: `ELSIF (x.type.form = OJB.Pointer) & (y.type.form = OJB.Pointer)`

Reply:
The new commit removes them.